### ISSUE TITLE
EncodeVarint() missing 1 required positional argument: 'unused_deterministic'

### ISSUE THREAD

**Issue by RoscaS** [krpc/krpc#416]
Same error on w10 and Ubuntu so I guess I'm doing something wrong. Anybody could give me a hand with this?

```
>>> import krpc
>>> conn = krpc.connect(name="test")
>>> vessel = conn.space_center.active_vessel
>>> print(vessel.name)
```

output:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/usr/local/lib/python3.5/dist-packages/krpc/client.py", line 85, in _invoke
    service, procedure, args, param_names, param_types, return_type)
  File "/usr/local/lib/python3.5/dist-packages/krpc/client.py", line 119, in _build_request
    request.arguments.add(position=i, value=Encoder.encode(value, typ))
  File "/usr/local/lib/python3.5/dist-packages/krpc/encoder.py", line 45, in encode
    return cls._encode_value(object_id, cls._types.as_type('uint64'))
  File "/usr/local/lib/python3.5/dist-packages/krpc/encoder.py", line 87, in _encode_value
    return getattr(_ValueEncoder, 'encode_' + typ.protobuf_type)(value)
  File "/usr/local/lib/python3.5/dist-packages/krpc/encoder.py", line 154, in encode_uint64
    return cls._encode_varint(value)
  File "/usr/local/lib/python3.5/dist-packages/krpc/encoder.py", line 123, in _encode_varint
    protobuf_encoder._VarintEncoder()(write, value)
TypeError: EncodeVarint() missing 1 required positional argument: 'unused_deterministic'
```

**Comment by djungelorm** [comment]
They added a new parameter to an internal method in protobuf v3.4.0 that the kRPC python client uses (unfortunately the required functionally isn't available via the public interface...) Downgrading to protobuf v3.3.0 should work around the issue until I update the python client.

**Comment by RoscaS** [comment]
Gosh, thanks. After searching for half the day on google and trying on different setups, I went trough all the answers of both your treads and finally found the answer on the middle of [this link](https://forum.kerbalspaceprogram.com/index.php?/topic/130742-130-krpc-control-the-game-using-c-c-java-lua-python-ruby-haskell-v039-14th-june-2017/&page=13).

I think it's the only place on the whole internet where this error shows up and it's not referenced by google. Now there is two places I guess... :)

After going trough all those discussions since the first iteration of the mod, I get to know you a bit and you seems to be such a good man ! Thanks for the amazing work !

**Comment by djungelorm** [comment]
Thanks - my pleasure :)

I've released python client v0.3.9.post1 on pypi that fixes this incompatibility. If you use pip to update your version of the package it should now work with the latest protobuf library: `pip install --upgrade krpc`
